### DISCUSSION TITLE
Notify only when detections change

### DISCUSSION THREAD

**Discussion by lkaino** [djdd87/SynoAI#178] [Ideas]
This is a really nice project, but I'm still getting a lot of notifications, when the lighting etc. changes and Surveillance Station is detecting this as movement.

It would be nice to utilize the AI a bit more and only send a notification when the detections change, e.g. number of cars or number of persons change. Even better would be if it could track the objects and only change notification if a new object is detected or lost.

**Comment by djdd87** [comment]
Hey thanks for the feedback. This is already logged in the issue tracker and someone has done a PR for it, but I've been too busy with other stuff to go through it and merge it in. I'm hoping to dedicate some time to catching up with this project in the next month or so.
